### PR TITLE
doc/use_bibtex

### PR DESCRIPTION
**Pull request by akucukelbir**
#### Summary:

+ introduces bibtex infrastructure to website
+ converts all pages to use appropriate cite commands

#### Intended Effect:

See above.

#### How to Verify:

Compile website. Look at pages.

#### Side Effects:

None.

#### Documentation:

None.

#### Reviewer Suggestions:

@dustinvtran

### CODE REVIEW

File: docs/tex/delving-in.tex [hunk 1]
```diff
@@ -169,11 +169,11 @@ \subsubsection{Criticism}\label{criticism}
...
 
-\begin{itemize}
-\item
-  Box, G.E.P. (1976). Science and Statistics. Journal of the American
-  Statistical Association, 71(356), 791–799
-\item
-  David M Blei. (2014). Build, compute, critique, repeat: Data analysis with
-  latent variable models. Annual Review of Statistics, 1:203-232
-\end{itemize}
+% \begin{itemize}
```
Comment (dustinvtran) [inline docs/tex/delving-in.tex]: should these comments be deleted like the rest?

**Comment by dustinvtran** [comment]
Cool! I like that it supports both `citep` and `citet` out of the box.

**Comment by dustinvtran** [comment]
Feel free to amend your single commit, deleting the lines from the comment above. I green-light merging after that.

File: docs/tex/tut_inference_networks.tex [hunk 2]
```diff
@@ -117,31 +116,9 @@ \subsubsection{Footnotes}
 general technique that can be applied to models beyond
 probabilistic decoders.
 
-This tutorial is taken from text originating in Tran et al. (2016).
+This tutorial is taken from text originating in \citet{tran2015variational}.
```
Comment (dustinvtran) [inline docs/tex/tut_inference_networks.tex]: actually it's tran2016variational. :) (the bib is [here](https://github.com/blei-lab/publications/blob/master/2016_TranRanganathBlei/TranRanganathBlei_2016.bib))

File: docs/tex/delving-in.tex (hunk 1 above)
Comment (akucukelbir) [inline docs/tex/delving-in.tex]: great catch

File: docs/tex/tut_inference_networks.tex (hunk 2 above)
Comment (akucukelbir) [inline docs/tex/tut_inference_networks.tex]: gah! google scholar caching issues. thank you.

**Pull request merged**